lib/src/builder/parsing/declarations_from_members.dart

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import 'package:analyzer/dart/ast/ast.dart';
```

Comment:
I don't really think that's necessary.
1. anything trying to use fully resolved AST just won't work, since we're not dealing with fully resolved AST
2. we don't do that anywhere else where we import this

Reply:
Because of other changes, this import is no longer needed